generators/app/index.js

```diff
@@ -88,6 +92,35 @@ module.exports = class extends Generator {
         message: 'Is this a form app?',
         default: false,
       },
+      {
+        type: 'input',
+        name: 'contentRepoLocation',
+        message:
+          'Where can I find the vagov-content repo? This path can be absolute or relative to vets-website. (Leave blank to skip this step.)',
+        default: () => {
+          const location = path.join(this.destinationRoot(), defaultContentRepoPath);
+          try {
+            fs.accessSync(location, fs.constants.F_OK | fs.constants.W_OK);
+            this.log(`Can access ${path.resolve(location)}`);
+            return path.resolve(location);
```

Comment:
just curious: by default this _will_ have a value, correct? And the user would have to manually clear it out to skip the step?

Reply:
Actually, this is a good point, now that you mention it. Yes and no. If we can find the `vagov-content` repo, "leaving it blank" will use the default of the path to the repo we found. If we couldn't find it, then leaving it blank will actually skip the step.

Like I said in the description, I was originally going to only ask this question if we couldn't find the repo, so "leave it blank" made sense at the time. With a default in there now, that's...misleading, isn't it? :thinking:

Maybe it doesn't make sense to skip this step after all; if they don't have `vagov-content`, they can't get to their app anyhow.

Alright, you've convinced me! I'll nix the "leave blank" bit. :+1: